src/ngraph/frontend/onnx_import/op/flatten.cpp

```diff
@@ -33,13 +33,10 @@ namespace ngraph
                     NodeVector inputs{node.get_ng_inputs()};
                     auto data = inputs.at(0);
                     auto axis = node.get_attribute_value<std::int64_t>("axis", 1);
+                    auto valid_axis =
+                        common::validate_axis(node, axis, data->get_shape().size() + 1);
```

Comment:
Why are you using `size() + 1` here?
If this is needed, it requires a line of documentation explaining it, or better yet a well named variable to hold this value: `data->get_shape().size() + 1`